modules/plottinggl/processors/parallelcoordinates/parallelcoordinatesaxissettingsproperty.h

```diff
@@ -0,0 +1,92 @@
+/*********************************************************************************
+ *
+ * Inviwo - Interactive Visualization Workshop
+ *
+ * Copyright (c) 2018 Inviwo Foundation
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *********************************************************************************/
+
+#ifndef IVW_PARALLELCOORDINATESAXISSETTINGSPROPERTY_H
+#define IVW_PARALLELCOORDINATESAXISSETTINGSPROPERTY_H
+
+#include <modules/plottinggl/plottingglmoduledefine.h>
+#include <inviwo/core/common/inviwo.h>
+#include <inviwo/core/properties/boolcompositeproperty.h>
+#include <inviwo/core/properties/minmaxproperty.h>
+#include <inviwo/core/properties/boolproperty.h>
+#include <modules/opengl/texture/texture2d.h>
+
+namespace inviwo {
+namespace plot {
+class Column;
+/**
+ * \class ParallelCoordinatesAxisSettingsProperty
+ * Helper class for handling axis specific tasks for the parallel coordinates plot
+ */
+class IVW_MODULE_PLOTTINGGL_API ParallelCoordinatesAxisSettingsProperty
+    : public BoolCompositeProperty {
+public:
+    friend class ParallelCoordinates;
+    InviwoPropertyInfo();
+    ParallelCoordinatesAxisSettingsProperty(std::string identifier, std::string displayName);
+    virtual ~ParallelCoordinatesAxisSettingsProperty() = default;
+
+    void updateFromColumn(std::shared_ptr<const Column> col);
+
+    double getNormalizedAt(size_t idx) const;
+    double getNormalized(double v) const;
+    double getValue(double v) const;
```

Comment:
documentation what this function returns. Why does it behave differently when usePercentiles is true?
Maybe renaming the function as well?